concourse/pipelines/bosh-cli/bosh-cli.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+
+image: docker:///governmentpaas/bosh-cli
+
+inputs:
+  - name: cf-manifest
+  - name: bosh-secrets
+run:
+  path: sh
+  args:
+  - -c
+  - -e
+  - |
+    bosh_password=$(awk '/bosh_admin_password/ { print $2 }' bosh-secrets/bosh-secrets.yml)
+    bosh -t 10.0.0.6 login admin "${bosh_password}"
+    bosh target 10.0.0.6
+    uuid=$(bosh status --uuid)
+    sed -e "s/^director_uuid:.*$/director_uuid: ${uuid}/" cf-manifest/cf-manifest.yml > cf-manifest-with-uuid.yml
```

Comment:
Alex and I discussed this. It looks like [it is `put` after the task](https://github.com/alphagov/paas-cf/blob/adac9f91590f86d950745c7c28d8838abef93c14/concourse/pipelines/deploy-cloudfoundry.yml#L502-L504), but `--inputs-from` takes the state from *before*. Nevermind.